huaweicloud/resource_huaweicloud_rds_instance_v3_test.go

```diff
@@ -92,22 +97,24 @@ func TestAccRdsInstanceV3_withEpsId(t *testing.T) {
 
 func testAccRdsInstanceV3_base(val string) string {
```

Comment:
suggest use more meaningful varibale name instead of `val`, such as `name` or `rName` which indicates the name of resource to be created

Reply:
updated.
These test have three name, there are `fmt.Sprintf("tf-acc-test-%s", acctest.RandString(5))`, `huaweicloud_rds_instance` and `huaweicloud_rds_instance.test`, do you have some suggestions for naming.

Reply:
naming of variables

Reply:
Usually we use the names `rName` and `resourceName`.
now, since the tests of `rds_read_replica_instance` and `rds_instance` share the existence check and destroy check methods, a new variable resource label like `huaweicloud_rds_instance` is introduced.